crates/backend/src/codegen.rs

```diff
@@ -167,7 +167,7 @@ impl ToTokens for ast::Struct {
                 }
             }
 
-            #[allow(clippy::all)]
+            #[allow(clippy::all, unsafe_code)]
```

Comment:
These look like unrelated changes?

Reply:
They definitely are; I added a few allows (`missing_docs` and `deprecated` for the `from_str`/`to_str` functions and `unsafe_code` wherever unsafe was used) but I can spin these off into their own PR or just remove them if they're problematic.

Reply:
Ah yeah I think it's best to have a dedicated PR for things like this if that's ok

Reply:
Sure, np. I'll make another PR